packages/boilerplates/BoilerplateDiscoverers.ts

```diff
@@ -0,0 +1,63 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Dolittle. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { BoilerplatePackageJson, ICanDiscoverBoilerplates, IBoilerplateDiscoverers } from './internal';
+
+/**
+ * Represents a system for dealing with all boilerplate discoverers
+ *
+ * @export
+ * @class BoilerplateDiscoverers
+ * @implements {IBoilerplateDiscoverers}
+ */
+export class BoilerplateDiscoverers implements IBoilerplateDiscoverers {
+    
+    /**
+     * Creates an instance of BoilerplateDiscoverers.
+     * @param {ICanDiscoverBoilerplates[]} boilerplateDiscoverers
+     * @memberof BoilerplateDiscoverers
+     */
+    constructor(boilerplateDiscoverers: ICanDiscoverBoilerplates[]) {
+        this.boilerplateDiscoverers = boilerplateDiscoverers;
+    }
+    
+    readonly boilerplateDiscoverers: ICanDiscoverBoilerplates[];
+    
+    /**
+     * @inheritdoc
+     *
+     * @readonly
+     * @type {string[]}
+     * @memberof BoilerplatesDiscoverer
+     */
+    get boilerplatePaths() {
+        let paths: string[] = [];
+        this.boilerplateDiscoverers.forEach(_ => paths.push(..._.boilerplatePaths));
+        return paths;
+    }
+    /**
+     * @inheritdoc
+     *
+     * @readonly
+     * @type {BoilerplatePackageJson[]}
+     * @memberof BoilerplatesDiscoverer
+     */
+    get discoveredBoilerplates(): BoilerplatePackageJson[] {
```

Comment:
No need for prefix; boilerplates is enough - it is in the discovererer and we're expecting discovered boilerplates from the context